dev/com.ibm.ws.classloading_test/bnd.bnd

```diff
@@ -33,16 +33,20 @@ test.project: true
 	com.ibm.ws.kernel.metatype.helper;version=latest,\
 	com.ibm.ws.kernel.security.thread;version=latest,\
 	com.ibm.ws.logging;version=latest,\
+	org.jmock:jmock-legacy;version=2.5.0, \
```

Comment:
Are these new dependencies required?  I would think that the existing JMock dependencies should be sufficient.

Reply:
There is a new mock to a class, not an interface, that requires this package when setting the mockery object with {
             {
                 setImposteriser(ClassImposteriser.INSTANCE);
             }
         };

Reply:
@teddyjtorres we need to stop using jmock-legacy, it will not work reliably with JDK 11